bin/run-semgrep.sh

```
@@ -0,0 +1,14 @@
+export SEMGREP_RULES="
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)